orderbook/openapi.yml

```diff
@@ -265,6 +291,29 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Trade:
+      description: |
+        Trade data such as executed amounts, fees, order id and block number.
+      type: object
+      properties:
+        blockNumber:
+          description: "Block in which trade occurred"
+          type: integer
+        logIndex:
+          description: "Index in which transaction was included in block"
+          type: integer
+        orderUid:
+          description: "Unique ID of the order matched by this trade"
+          $ref: "#/components/schemas/UID"
+        sellAmount:
+          description: "Amount sold in this trade"
+          $ref: "#/components/schemas/BigUint"
+        buyAmount:
+          description: "Amount bought in this trade"
+          $ref: "#/components/schemas/BigUint"
+        feeAmount:
+          description: "Fee paid (in sell token)"
+          $ref: "#/components/schemas/BigUint"
```

Comment:
For these the type can be `TokenAmount` instead of `BigUint` because they are limited to `U256`.